source/plugin/commands/arguments.rst

```diff
@@ -25,13 +25,15 @@ it with :javadoc:`CommandContext#getOne(String)`. Optional and weak arguments ma
 
 Many of the parsers may return more than one object (e.g. multiple players with a matching username). In that case, you
 must use the :javadoc:`CommandContext#getAll(String)` method to get the ``Collection`` of possible matches.
-**Otherwise, the context object will throw an exception!**
+**Otherwise, the context object will throw an exception!** Please make sure that you make a conscious decision whether
```

Comment:
Not sure it needs to be this wordy. I think "You must decide whether you …" should do.
It also needs a comma and some word tweaking later "... single result, or allow it to support multiple arguments."

Reply:
This should be covered by dualspiral's suggestion